### DISCUSSION TITLE
UP主的页面太简陋了，不全面，看起来不太舒服（个人意见）

### DISCUSSION THREAD

**Discussion by Q-Tai-mu** [Richasy/Bili.Uwp#723] [展示]
现在的up主页面我感觉太简陋了
![image](https://user-images.githubusercontent.com/58099119/147812594-f135850e-006b-4f54-8d40-882010e558af.png)
我稍微提供下自己的想法
![1640939487(1)](https://user-images.githubusercontent.com/58099119/147812558-bd6ac3fd-7450-4f93-8c9a-9fb61cc1af1b.png)

**Comment by xvzhenduo** [comment]
你应该发到issue里面的，这样更容易让开发者看到

**Reply by Q-Tai-mu** [reply]
我的小意见，哈哈哈，没必要吧
